packages/tic-tac-toe/.env

```diff
@@ -33,4 +33,4 @@ INFURA_API_KEY='A valid Infura API key is required to deploy to a test network'
 
 # These contract addresses get defined in the start/build scripts 
 # or the contract test setup with the actual addresses of the contracts
-TTT_CONTRACT_ADDRESS=0x0000000000000000000000000000000000000000
+TTT_CONTRACT_ADDRESS=0x430869383d611bBB1ce7Ca207024E7901bC26b40
```

Comment:
This should probably be set in `.env.production` if this is a Ropsten address. Maybe also indicate the git commit has at which the contract was deployed from. Alternatively, verify the source code on etherscan.

Reply:
I haven't deployed to Ropsten yet. Can you teach me how we do this?

Reply:
What is this address then? You can probably use Truffle or Etherlime to deploy. I used `deploy.js` but modified the `GanacheDeployer` to one used for a real network when I did  it. I did not upload that code to version control.

Reply:
This is the hard coded address from the local Gamache deploy. I had issues injecting the actual address during development so hard coding it for now so I don't get blocked.

Reply:
@Alonski If you're following the same pattern as `rps` by the way this should be getting auto-injected by the deploy script.

Reply:
If that's not working, I recommend putting this value into `.env.${NODE_ENV}.local` so it is not in version control.